docs/Options.md

```diff
@@ -180,86 +239,69 @@ Values: `true` or `false` | Default =  `false`
 
 Set uWSGI [log-x-forwarded-for](https://uwsgi-docs.readthedocs.io/en/latest/Options.html?#log-x-forwarded-for) parameter allowing to keep the real IP of clients in logs when using a reverse proxy.
 
-### `LD_DB_ENGINE`
-
-Values: `postgres` or `sqlite` | Default = `sqlite`
-
-Database engine used by linkding to store data.
-Currently, linkding supports SQLite and PostgreSQL.
-By default, linkding uses SQLite, for which you don't need to configure anything.
-All the other database variables below are only required for configured PostgresSQL.
-
-### `LD_DB_DATABASE`
 
-Values: `String` | Default =  `linkding`
+## Single File Options
 
-The name of the database. 
 
-### `LD_DB_USER`
+### `LD_ENABLE_SNAPSHOTS`
```

Comment:
This should not be exposed to users, it's only an internal marker for the app to know whether the dependencies for creating snapshots are installed and can be used. Changing this in the base image would result in broken features. Let's remove this.